rockcraft/project.py

```diff
@@ -144,6 +145,41 @@ def _validate_platform_set(cls, values: Mapping[str, Any]) -> Mapping[str, Any]:
         return values
 
 
+class Service(pydantic.BaseModel):
+    """Lightweight schema validation for a Pebble service.
+
+    Based on
+    https://github.com/canonical/pebble#layer-specification
+    """
+
+    override: Literal["merge", "replace"]
+    command: str
+    summary: Optional[str]
+    description: Optional[str]
+    startup: Optional[Literal["enabled", "disabled"]]
+    after: Optional[List[str]]
+    before: Optional[List[str]]
+    requires: Optional[List[str]]
+    environment: Optional[Dict[str, str]]
+    user: Optional[str]
+    user_id: Optional[int] = pydantic.Field(alias="user-id")
```

Comment:
A model-wide alias can be defined to replace underscores instead of doing it in each field (see `alias_generator`).